curator-framework/src/main/java/org/apache/curator/framework/imps/CreateBuilderImpl.java

```diff
@@ -83,6 +85,13 @@ public CreateBuilderMain orSetData()
         return this;
     }
 
+    @Override
+    public CreateBuilderMain withTtl(long ttl)
```

Comment:
I prefer withTTL, but that's just personal preference. I'm not sure what the standard for capitalisation of acronyms is.